skyportal/models/phot_stat.py

```diff
@@ -71,9 +71,9 @@ def __init__(self, obj_id):
         self.mag_rms_global = None
         self.mag_rms_per_filter = {}
 
-    read = public
+    read = update = public
 
-    write = update = delete = restricted
+    write = delete = restricted
```

Comment:
Why is this? Is it because you can't update upon delete/add a photometry point?

Reply:
Yes.